README.md

```diff
@@ -202,28 +208,27 @@ corresponding exercises.
 
 ## Set up a project in an IDE `[LO-IdeSetup]`
 
-#### Exercise: Setup project in Eclipse 
+#### Exercise: Setup project in IntelliJ 
 
-* Learn [how to set up a new project in Eclipse]
-  (https://se-edu.github.io/addressbook-level1/doc/Getting started with Eclipse.pptx). <br>
-  Create a new project in Eclipse and write a small HelloWorld program.
+* Learn [how to set up a new project in IntelliJ (from jetbrains.com)]
+  (https://www.jetbrains.com/help/idea/creating-running-and-packaging-your-first-java-application.html). <br>
```

Comment:
there seems to be an unintended error here (it was present in the original file as well), but I think we can fix it ba. Seems weird to have square brackets at `[how to set up a new project in IntelliJ (from jetbrains.com)]`